examples/to-do/index.es

```diff
@@ -0,0 +1,33 @@
+Element `to-do`
+
+(class extends HTMLElement {
+
+  initialize () {
+    this.context
+      .tasks = ['Wash clothes', 'Eat food']
+  }
+
+  onsubmit (event, input = this.select `input`) {
+    event.preventDefault ()
+
+    this.context
+      .tasks.push (input.value)
+
+    input.value = ''
+  }
+
+  remove (event) {
+    this.context.tasks
+      .splice (event.target.id, 1)
+  }
+
+  get tasks ()
+    { return this.context.tasks }
+
+  get name ()
+    { return 'Loren Hale' }
+
+  get count ()
+    { return this.context.tasks.length }
+})
+
```

Comment:
You mentioned a blank line at the bottom of source files and mentioned that they're for me.

I don't like blank lines at the bottom. I think you're thinking of no line ending on the last line, which I think is horrible. 😄